source/mainnet/docs/network/guides/validation-with-wallets.rst

```diff
@@ -0,0 +1,142 @@
+.. include:: ../../../variables.rst
+
+.. _validation-with-wallets:
+
+==================================
+Validation with Concordium wallets
+==================================
+
+
+Validation is possible with the Concordium wallets as well as with the ``Concordium-client``, however the process differs between them. The overviews below give a brief description of the process.
+
+.. Attention::
+
+   Before proceeding, read :ref:`Validator management<baker-pool>` for information about best practices for validators.
+
+.. Note::
+
+   To check the minimum required amount of CCD (currently 500000) to become a validator, see :ref:`consensus show-chain-parameters`.
+
+Validation with |bw|
+------------------------
+
+This overview describes the recommended scenario for running a node and becoming a validator on the Concordium blockchain when using |bw| and running a node.
```

Comment:
PR was recently opened on this guide, I merged 2 sections together because there was a lot of duplicate information, I'm waiting for review. Here's https://github.com/Concordium/concordium.github.io/pull/1141 for reference.

Reply:
ok, can we review/merge the above PR. Let me know when this PR is ready for review :).